BUILDING.md

```diff
@@ -24,6 +24,7 @@ file a new issue.
     * [Unix prerequisites](#unix-prerequisites)
     * [macOS prerequisites](#macos-prerequisites)
     * [Building Node.js](#building-nodejs-1)
+    * [Installing Node.js](#installing-nodejs-1)
```

Comment:
```suggestion
    * [Installing Node.js](#installing-nodejs)
```

This is also in the wrong place, as the actual heading has been inserted between "Building the documentation" and "Building a debug build".

Reply:
@richardlau - fixed. please have a look.